util/formula_templater/config.hcl

```diff
@@ -0,0 +1,190 @@
+
+formula {
+    product = "boundary"
+    name = "Boundary"
+    desc = "Boundary"
+    homepage = "https://www.boundaryproject.io/"
+    architectures {
+        darwin_amd64 = true
+        darwin_arm64 = false
+        linux_amd64 = true
+        linux_arm = true
+        linux_arm64 = true
+    }
+}
+
+formula {
+    product = "consul"
+    name = "Consul"
+    desc = "Consul"
+    homepage = "https://www.consul.io"
+    architectures {
+        darwin_amd64 = true
+        darwin_arm64 = false
+        linux_amd64 = true
+        linux_arm = false
+        linux_arm64 = true
+    }
+    plist =<<EOF
```

Comment:
Noob question, but would adding property lists with standarderror, etc be useful for waypoint/boundary/terraform as well?

Reply:
Honestly I have no idea 😂 But since this is supported then the teams or community can PR this if it seems useful